go.mod

```diff
@@ -6,6 +6,7 @@ go 1.13
 
 require (
 	github.com/gobuffalo/attrs v1.0.0
+	github.com/gobuffalo/envy v1.9.0 // indirect
```

Comment:
I feel like this needs a `go mod tidy`. :)

Reply:
This one actually came in because I ran go mod tidy. I did ran `go mod why` and it seems pop v5 depends on it.

```
# github.com/gobuffalo/envy
github.com/gobuffalo/buffalo-cli/v2/cli/internal/plugins/fizz/generate
github.com/gobuffalo/pop/v5
github.com/gobuffalo/envy
```